test/integration/default/virtualization_spec.rb

```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
+if ENV['DOCKER']
```

Comment:
We could simplify this to:

```
if ENV['DOCKER'] && os.linux
   # rest of code here
end
```

... and eliminate one indentation level. Not critical to fix here, just something for the future :)